### PR TITLE
fix: replace imgur with s3 for arabic guide files with no conflict

### PR DESCRIPTION
**Pull request by ahmaxed**
(cherry picked from commit 96a14a1f278c12e0123216813ab27076b1bcdb66)

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Reference #36007

### CODE REVIEW

File: guide/arabic/cplusplus/tokens-variables/index.md [hunk 1]
```diff
@@ -21,8 +22,8 @@ localeTitle: توكنز الجزء الأول
 ### ما هي المتغيرات؟
 
 *   تعريف الكتب: تُسمى المتغيرات أسماء الذاكرة التي يمكن تغيير بياناتها.
-    
-*   لكنني أود منك أن تفكر في متغير ليصبح شيئًا مثل مربع ، شيء من هذا القبيل: ![IMG](https://i.imgur.com/YdbgWHL.png)
+    https://cdn-media-1.freecodecamp.org/imgr/
```
Comment (RandellDawson) [inline guide/arabic/cplusplus/tokens-variables/index.md]: Is this line a mistake?
```suggestion
```

File: guide/arabic/cplusplus/tokens-variables/index.md [hunk 2]
```diff
@@ -170,8 +171,8 @@ localeTitle: توكنز الجزء الأول
 إذا لم يتم تهيئة متغير ، فإنه يحتوي على قيمة للقمامة. فمثلا:
 
 لذلك من حيث المربعات ، يمكنك تخيل هذا على أنه -
-
-![IMG](https://i.imgur.com/YdbgWHL.png)
+https://cdn-media-1.freecodecamp.org/imgr/
```
Comment (RandellDawson) [inline guide/arabic/cplusplus/tokens-variables/index.md]: Is this line a mistake?
```suggestion
```

**Review by RandellDawson — CHANGES_REQUESTED** [review]
@ahmadabdolsaheb See my two suggested changes.

**Comment by ahmadabdolsaheb** [comment]
@RandellDawson thanks for the suggestions. It should be alright now.

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
@ahmadabdolsaheb OK, this PR looks good now.

**Comment by ahmadabdolsaheb** [comment]
I edited some spaces out, this should be good to go

File: guide/arabic/css/properties/background-position-property/index.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ localeTitle: موقع الخلفية الملكية
 
  `/* setting background-image of HTML doc */ 
  body { 
-  background-image: url('https://i.imgur.com/6Z2VStD.png'); 
+  background-image: url('https://cdn-media-1.freecodecamp.org/imgr/6Z2VStD.png');
```
Comment (thecodingaviator) [inline guide/arabic/css/properties/background-position-property/index.md]: Referring just so we don't forget this in the other PR we've discussed

**Review by thecodingaviator — APPROVED** [review]
LGTM

**Comment by thecodingaviator** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Pull request merged**